src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.data.person;
+
+
+// Creates a Block to take in Block variable
```

Comment:
Not a correct format for header comment and no need to mention `Block variable`.. Check how others name their classes.